### ISSUE TITLE
ORM compilation bug when using multiple databases of different types such as sqlite and postgres

### ISSUE THREAD

**Issue by saasira** [vlang/v#22584]
### Describe the bug

when using different databases such as sqlite and postgres in the same program, and attempting to assign the last inserted id to another variable, we are getting compilation error
_error converting between sqlite.DB and pg.DB_

if we do not assign the return value of ORM insertion operation to anything, then the program compiles fine without issues.

### Reproduction Steps

```v
import db.pg
import db.sqlite

fn main() {
        dbpg := pg.connect(pg.Config{
		host:'127.0.0.1',
		port:5432,
		user:'postgres',
		password:'postgres',
		dbname:'addresses'
	}) or { panic(err)}
	
	defer {
		dbpg.close()
	}

	println("connected to postgres database")

	dblite_path := '../data/addresses.db'
	mut dblite := sqlite.DB {}
	if os.exists(dblite_path) {
		dblite = sqlite.connect(dblite_path) or { panic(err) }	
	} else {
		dblite = sqlite.connect_full(dblite_path, [sqlite.OpenModeFlag.create], "default")!
		sql dblite {
			create table Address
		}!
	}
	
	address := Address {
	        city : 'Vijayawada'
	        state : 'Andhra'
		country : 'IND'
	}
	//address_id := sql dblite {         //this fails with compilation error converting between db_sqlite_DB and db_pg_DB
	sql dblite {    //compilation works fine 
		insert address into Address
	} or {
		println("error inserting address")
		panic(err)
	}
}
```

### Expected Behavior

should be able to use V ORM with multiple database references, connecting to different database systems such as sqlite, postgres, mysql, etc in the program/function

### Current Behavior

getting compilation error "cannot convert 'struct db__sqlite__DB' to 'struct db__pg__DB'" when using db.sqlite and db.pg drivers in the same program

### V version

V 0.4.8 d1b0370

### Environment details (OS name and version, etc.)

Windows 10

> [!NOTE]
> You can use the 👍 reaction to increase the issue's priority for developers.
>
> Please note that only the 👍 reaction to the issue itself counts as a vote.
> Other reactions and those to comments will not be taken into account.

**Comment by medvednikov** [comment]
It's not due to using different databases. The returninggID thing is only implemented for pg for now.

**Comment by spytheman** [comment]
This compiles for me, with latest V:
```v
import os
import db.pg
import db.sqlite

struct Address {
    city string
    state string
    country string
}

fn main() {
    dbpg := pg.connect(host:'127.0.0.1', port:5432,	user:'postgres', password:'postgres', dbname:'addresses')!
	defer {	dbpg.close() }

	println("connected to postgres database")

	dblite_path := '../data/addresses.db'
    
    mut dblite := sqlite.DB{}
	if os.exists(dblite_path) {
		dblite = sqlite.connect(dblite_path)!
	} else {
		dblite = sqlite.connect_full(dblite_path, [sqlite.OpenModeFlag.create], "default")!
		sql dblite {
			create table Address
		}!
	}	
	
    address := Address {
	        city : 'Vijayawada'
	        state : 'Andhra'
	     	country : 'IND'
    }
    println(address)
    address_id := sql dblite {
	insert address into Address
    } or {
	println("error inserting address")
	panic(err)
    }
    println(address_id)
}
```

**Comment by spytheman** [comment]
Fixed in latest V after #22582 .

**Comment by medvednikov** [comment]
Great, thanks!
